### PR TITLE
Add pascals-triangle problem to erlang

### PR DESCRIPTION
**Pull request by trungda**
Add a new problem: pascals-triangle.
I ran `rebar3 eunit` in my local machine and all test cases passed.

```
pascals-triangle $ rebar3 eunit
===> Verifying dependencies...
===> Compiling pascals_triangle
src/pascals_triangle.erl:5: Warning: variable 'N' is unused

===> Performing EUnit tests...
======================== EUnit ========================
file "pascals_triangle.app"
  application 'pascals_triangle'
    module 'example'
    module 'pascals_triangle'
      module 'pascals_triangle_tests'
        pascals_triangle_tests: zero_row_test...ok
        pascals_triangle_tests: one_row_test...ok
        pascals_triangle_tests: two_rows_test...ok
        pascals_triangle_tests: three_rows_test...ok
        pascals_triangle_tests: four_rows_test...ok
        pascals_triangle_tests: five_rows_test...ok
        pascals_triangle_tests: six_rows_test...ok
        pascals_triangle_tests: ten_rows_test...ok
        pascals_triangle_tests: negative_rows_test...ok
        [done in 0.027 s]
      [done in 0.027 s]
    [done in 0.036 s]
  [done in 0.045 s]
=======================================================
  All 9 tests passed.

```

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -451,7 +451,15 @@
...
 
       ]
-    }
+    },
+    {
+      "uuid": "589b763c-2b3a-11e8-b467-0ed5f89f718b",
+      "slug": "pascals-triangle",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 6,
```
Comment (NobbZ) [inline config.json]: Can you put this somewhere near other exercises with difficulty `6`?

Comment (trungda) [reply config.json]: I just did :) Could you merge this?

Comment (trungda) [reply config.json]: @NobbZ Could you merge this PR?

Comment (NobbZ) [reply config.json]: Sorry for the delay...

Its quite busy here ;)

I'll merge.

**Pull request merged**